documentation/course/E1.md

```diff
@@ -612,16 +627,89 @@ TODO@piterek130
 
 ### Wymaganie 7
 
-TODO@mlodybercik
+- `LOG/02` - Jako _administrator_ chcę mieć możliwość przeglądania, dodawania, edycji i usuwania _przystanków_, aby zarządzać miejscami, w których _pasażerowie_ mogą wsiąść lub wysiąść z _pojazdów_.
 
 #### Mapa nawigacyjna
 
+```mermaid
+classDiagram
+    direction LR
+    class Logowanie { <<serverPage>> }
+    class MenuAdministratora { <<serverPage>> }
+    class ZarzadzaniePrzystankami { <<serverPage>> }
+    class EdycjaPrzystanku { <<modalPopup>> }
+    class UsuwaniePrzystanku { <<modalPopup>> }
+
+    Logowanie --> MenuAdministratora
+    MenuAdministratora --> ZarzadzaniePrzystankami : btnPrzystanki
+    ZarzadzaniePrzystankami --> EdycjaPrzystanku : btnEdytuj
+    EdycjaPrzystanku --> ZarzadzaniePrzystankami : btnZapisz
+    EdycjaPrzystanku --> ZarzadzaniePrzystankami : btnAnuluj
+
+    ZarzadzaniePrzystankami --> UsuwaniePrzystanku : btnUsuń
+    UsuwaniePrzystanku --> ZarzadzaniePrzystankami : btnTak
+    UsuwaniePrzystanku --> ZarzadzaniePrzystankami : btnAnuluj
+
+    EdycjaPrzystanku --> UsuwaniePrzystanku : btnUsuń
+    UsuwaniePrzystanku --> EdycjaPrzystanku : btnAnuluj
+    UsuwaniePrzystanku --> ZarzadzaniePrzystankami : btnTak
+```
+
 #### Prototypy ekranów
 
+- `LOG/02/01` - Przeglądanie listy przystanków.
+
+![LOG/02/01](images/Stop1.png)
+
+- `LOG/02/02` - Dodawanie przystanku.
+
+![LOG/02/02](images/Stop2.png)
+
+- `LOG/02/03` - Edycja przystanku.
+
+![LOG/02/03](images/Stop3.png)
+
+- `LOG/02/04` - Usuwanie przystanku.
+
+![LOG/02/04](images/Stop4.png)
+
 ### Wymaganie 8
 
-TODO@mlodybercik
+- `LOG/06` - Jako _bileter_ chcę mieć możliwość przeglądania, zgłaszania i edycji _awarii_, aby informować _pasażerów_ o problemach na _kursie_.
 
 #### Mapa nawigacyjna
 
```

Comment:
Let's keep the formatting consistent with @jakubzehner. I don't really care which format we will end up using, but currently we are using two clashing standards.

I'd even say that your formatting looks better than Kuba's. In that case you might change how requirements are written down in his part.

I'd also write down your name next to the requirement, so that it will be easier for our instructor to check these, since they are treated as individual work.

Reply:
I agree

Reply:
I assumed that for simplicity, every one of us would take tightly-linked requirements for UI prototyping, thus not requiring multiple navigational maps. @jakubzehner has 2 + 1 + 1 by the looks of it, so my formatting as is, will not cut it. I'll try to come up with something that will work for us both.